src/screens/DiscoveryScreenV2/fragment/NewsFragment.js

```diff
@@ -1,161 +1,140 @@
 import * as React from 'react';
-import {StyleSheet, Text, View} from 'react-native';
-import {useNavigation} from '@react-navigation/native';
+import { StyleSheet, Text, View } from 'react-native'
+import { useNavigation } from '@react-navigation/native';
 
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import LoadingWithoutModal from '../../../components/LoadingWithoutModal';
 import RecentSearch from '../elements/RecentSearch';
 import RenderItem from '../elements/RenderItem';
 import share from '../../../utils/share';
 import useIsReady from '../../../hooks/useIsReady';
-import {Context} from '../../../context/Store';
-import {fonts} from '../../../utils/fonts';
-import {getUserId} from '../../../utils/users';
-import {newsDiscoveryContentParamBuilder} from '../../../utils/navigation/paramBuilder';
-import {withInteractionsManaged} from '../../../components/WithInteractionManaged';
-import {COLORS} from '../../../utils/theme';
+import { Context } from '../../../context/Store'
+import { colors } from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated. Ensure that the `colors` module exports the required color properties and that they are correctly referenced throughout the codebase.
```diff
- import {COLORS} from '../../../utils/theme';
+ import {colors} from '../../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->